tests/test_job_management.py

```diff
@@ -90,6 +90,73 @@ def test_public_async_request(dispatcher_live_fixture, dispatcher_local_mail_ser
 @pytest.mark.parametrize("default_values", [(True), False])
 @pytest.mark.parametrize("time_request_none", [True, False])
 def test_email_callback_after_run_analysis(dispatcher_live_fixture, dispatcher_local_mail_server, default_values, time_request_none):
+#def test_time_request(dispatcher_live_fixture, dispatcher_local_mail_server): #VS: sorry, was not sure what to do with name? 
```

Comment:
I had made to separate test to test the two parameters separately. But better in this way, more testing has been actually performed. I change also the way the time of the request was initialized